src/Formio.js

```diff
@@ -515,7 +515,7 @@ export default class Formio {
     });
   }
 
-  uploadFile(storage, file, fileName, dir, progressCallback, url, options, fileKey) {
```

Comment:
We cannot change the signature of this function without a major version release. Just append the arguments to the end please.

Reply:
@travist, I moved the argument to the beginning, since the number of arguments varies and breaks existing logic